Makefile.validation

```diff
@@ -140,6 +145,9 @@ kind-install: kind-build
 kind-create-cluster:
 	$(KIND) create cluster --config $(KIND_DIR)/kind.yaml --image kindest/node:v1.18.2
 
+kind-create-cluster-v19:
```

Comment:
Maybe better to have target with env variables? 
Something like this:
```
kind-create-cluster:
	$(KIND) create cluster --config $(KIND_DIR)/$(KIND_CONFIG) --image kindest/node:$(KIND_IMAGE_VERSION)
```

Reply:
Ok, but i also add `--wait` to this target and set default value for it. Hope this is not critical...

Reply:
No, it breaks our pipeline, don't put `--wait` to Makefile, use outside command.